web/testing/issue005/header.js

```diff
@@ -0,0 +1,129 @@
+    The actual keyboard object will be downloaded asynchronously when first selected for use.
+  
+    Each argument to addKeyboards() is a string, for example:
+      european2         loads the current version of the Eurolatin 2 keyboard (for its default language)
+      european2@fra     loads the current version of the Eurolatin 2 keyboard for French
+      european2@fra@1.2 loads version 1.2 of the Eurolatin 2 keyboard for French
+      
+    Argument syntax also supports the following extensions:
+      @fra              load the current version of the default keyboard for French
+      @fra$             load all available keyboards (current version) for French
+          
+    Each call to addKeyboards() requires a single call to the remote server, 
+    (unless all keyboards listed are local and fully specified) so it is better
+    to use multiple arguments rather than separate function calls. 
+    
+    Calling addKeyboards() with no arguments returns a list of *all* available keyboards. 
+    The Toolbar (desktop browser) UI is best suited for allowing users to select 
+    the appropriate language and keyboard in this case.
+
+    Keyboards may also be specified by language name using addKeyboardsForLanguage()
+    for example:
+      keymanweb.addKeyboardsForLanguage('Burmese');
+    
+    Appending $ to the language name will again cause all available keyboards for that
+    language to be loaded rather than the default keyboard.
+        
+    The first call to addKeyboardsForLanguage() makes an additional call to the 
+    keyman API to load the current list of keyboard/language associations.
+
+    In this example, the following function loads the indicated keyboards,
+    and is called when the page loads. 
+*/
+
+  function loadKeyboards() 
+  { 
+    var kmw=window['keyman'] ? keyman : tavultesoft.keymanweb;
+    
+    // The first keyboard added will be the default keyboard for touch devices.
+    // For faster loading, it may be best for the default keybaord to be 
+    // locally sourced.
+    kmw.addKeyboards({id:'us',name:'English',languages:{id:'eng',name:'English'},
+      filename:'../us-1.0.js'});
+      
+    // Add more keyboards to the language menu, by keyboard name,
+    // keyboard name and language code, or just the ISO 639 language code.  
+    kmw.addKeyboards('french','european2@swe','european2@nor','@heb');
+  
+    // Add a keyboard by language name.  Note that the name must be spelled
+    // correctly, or the keyboard will not be found.  (Using ISO codes is
+    // usually easier.)
+    kmw.addKeyboardsForLanguage('Dzongkha');
+    
+    // Add a fully-specified, locally-sourced, keyboard with custom font  
+    kmw.addKeyboards({id:'lao_2008_basic',name:'Lao Basic',
+      languages:{
+        id:'lao',name:'Lao',region:'Asia',
+        font:{family:'LaoWeb',source:['../font/saysettha_web.ttf','../font/saysettha_web.woff','../font/saysettha_web.eot']}
```

Comment:
Do you have a local /testing/fonts folder?
My chrome console is not finding /testing/fonts/saysettha_web.*

Reply:
Nope, that issue's been there for quite a while; it's something I've just simply lived with for now.  It's straight from the samples-section samplehdr.js file.